test/Index/index_objc_subscript.swift

```diff
@@ -0,0 +1,35 @@
+// REQUIRES: objc_interop
+
+// RUN: %empty-directory(%t)
+// RUN: split-file %s %t
+
+// RUN: %target-swift-ide-test -print-indexed-symbols -source-filename %t/SplitSubscript.swift -import-objc-header %t/objc_impl.h > %t/output.txt
+// RUN: %FileCheck %s -input-file %t/output.txt
+
```

Comment:
Thanks @QuietMisdreavus! Is there an existing test for non-split subscripts? If not, could you add it here?

> When importing subscript methods from Objective-C, we currently use the Clang node's USR as the USR for the subscript decl as a whole. However, in the following code, both classes' subscripts receive the same USR

Which decl does it use for the subscript decl? From NOT below I assume it was the getter?

Reply:
Right now, the USR for the subscript as a whole is taken from the getter. This PR changes it to be taken from the subscript decl itself, which in this case is the one generated by the Clang importer.